### PR TITLE
EZP-25661: Default columns in the subitem list view

### PR DESCRIPTION
**Pull request by dpobel**
JIRA: https://jira.ez.no/browse/EZP-25661

# Description

The patch changes the sub-item list to display the following columns by default:

* Name
* Modified
* Content type
* Priority
* Translations

The patch is quite big because it also brings a flexible system so that we can easily support others columns and then let the user to choose which columns to display.

[](http://www.youtube.com/watch?v=ENd8vgnLV5c "Click to play on Youtube.com")

# Tests

unit tests and manual test

### CODE REVIEW

**Comment by dpobel** [comment]
ping @StephaneDiot @yannickroger

File: Resources/public/js/views/subitem/ez-subitemlistitemview.js [hunk 1]
```diff
@@ -0,0 +1,504 @@
...
+         * @method _getContentTypeName
+         * @protected
+         * @return {String}
+         */
+        _getContentTypeName: function () {
+            return this.get('contentType').get('names')['eng-GB'];
+        },
+
+        /**
+         * Returns an attribut valuee of the Location
```
Comment (yannickroger) [inline Resources/public/js/views/subitem/ez-subitemlistitemview.js]: Typo: seems an `e` jumped at the end of the next word

**Comment by yannickroger** [comment]
+1

**Comment by StephaneDiot** [comment]
+1

**Pull request merged**